# <center> **DSCI 100 Project: Predicting Usage of a Video Game Server**
# <center> **Final Report**

In [1]:
library(tidyverse)
library(repr)
library(tidymodels)
options(repr.matrix.max.rows = 6)

── Attaching core tidyverse packages ──────────────────────── tidyverse 2.0.0 ──
✔ dplyr     1.1.4     ✔ readr     2.1.5
✔ forcats   1.0.0     ✔ stringr   1.5.1
✔ ggplot2   3.5.1     ✔ tibble    3.2.1
✔ lubridate 1.9.3     ✔ tidyr     1.3.1
✔ purrr     1.0.2     
── Conflicts ────────────────────────────────────────── tidyverse_conflicts() ──
✖ dplyr::filter() masks stats::filter()
✖ dplyr::lag()    masks stats::lag()
ℹ Use the conflicted package (<http://conflicted.r-lib.org/>) to force all conflicts to become errors
── Attaching packages ────────────────────────────────────── tidymodels 1.1.1 ──

✔ broom        1.0.6     ✔ rsample      1.2.1
✔ dials        1.3.0     ✔ tune         1.1.2
✔ infer        1.0.7     ✔ workflows    1.1.4
✔ modeldata    1.4.0     ✔ workflowsets 1.0.1
✔ parsnip      1.2.1     ✔ yardstick    1.3.1
✔ recipes      1.1.0     

── Conflicts ───────────────────────────────────────── tidymodels_conflicts() ──
✖ scales::discard() masks purrr::discard()
✖ dplyr::filt

## Introduction

Question: Can age and amount of time playing video games predict whether or not a player has subscribed to the gaming newsletter?

Description of the Players dataset:

The dataset consists of data for MineCraft video game players of different ages with varying levels of experience in gaming and number of hours played. The logistics of their name, gender, (age), email and whether they have subscribed to the newsletter have been recorded. There are 7 variables and 196 observations in the dataset.

Outcome Variable: The main outcome/categorical/label variable in the dataset is the 'subscribe' variable which describes whether or not a player has subscribed to the Minecraft newsletter. Thus, there are two groups within the category: "TRUE" or "FALSE". We can double check using the distinct function below.

In [4]:
players |>
    distinct(subscribe)

subscribe
1  TRUE    
2 FALSE

To determine the number of observations in each group of "TRUE" or "FALSE", we can group the observations according to the 'subscribe' variable and summarize the number of counts or observations in each group.

In [5]:
num_subscribe <- players |>
    group_by(subscribe) |>
    summarize(count = n())
num_subscribe

subscribe count
1 FALSE      52  
2  TRUE     144

From the tibble above, we can see that there are 144 players who have subscribed to the gaming newsletter but 52 players who have not done so yet.

## Methods

## Results

## Discussion

## References